portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/stylebooks/StyleBooks.testcase

```diff
@@ -124,6 +124,8 @@ definition {
 	@description = "This is for LPS-116078. Can not preview style book when there are no pages in site."
 	@priority = "4"
 	test CannotPreviewStyleBookWhenSiteWithoutPage {
+		property portal.acceptance = "true";
+
```

Comment:
Remove this. Don't need to add more of their tests onto acceptance. We'll let Echo team decide what they want.

Reply:
Add `property testray.component.names = "Frontend Token";` instead and filter this in test.properties